app/web/src/components/Workspace/WorkspaceModelAndView.vue

```diff
@@ -177,8 +187,108 @@ import ComponentCard from "../ComponentCard.vue";
 import EdgeCard from "../EdgeCard.vue";
 import NoSelectionDetailsPanel from "../NoSelectionDetailsPanel.vue";
 
+const realtimeStore = useRealtimeStore();
 const changeSetStore = useChangeSetsStore();
 const fixesStore = useFixesStore();
+const authStore = useAuthStore();
+
+const MOUSE_REFRESH_RATE = 20;
```

Comment:
I'd probably move most of this logic to a store

Reply:
I would wait to do that until we want to extend on this. If this logic gets bigger or other places have to replicate it. It will also guide us on the best way to do it.

Doesn't feel required for this first iteration.